components/Header/index.js

```diff
@@ -31,18 +40,16 @@ export default function Header() {
 						router.push("/maps");
 						closeModal();
 					}}
-					variant="contained"
-					sx={{fontSize: 16}}>
-					<span>Home</span>
-					<Icon.Home sx={{fontSize: 20}} />
+					variant="text"
+					color="inherit">
+					<Icon.Home sx={{fontSize: 36}} />
+					<span>Dashboard</span>
```

Comment:
What is this `<span>` used for here? If it's simply the text of the button, you don't need the span.
I've only found a use case for this [when it's a loading button](https://mui.com/material-ui/react-button/#loading-button).

Besides, does the `Button` has it's type specified or is it a (default) submit button?